### ISSUE TITLE
NVDA doesn't announce current cursor position in centimetres from the left/top edge of the page

### ISSUE THREAD

**Issue by peterpacult** [nvaccess/nvda#12176]
### Steps to reproduce:
1. Start NVDA and LO Writer
2. Press NVDA+Delete
3. NVDA will announce numbers, but it is not clear what these do exactly mean; it is at least not the distance from the left and top margins in centimetres

### Actual behavior:
NVDA does not read the distance from the left and top margins in centimetres

### Expected behavior:
NVDA should read the distance from the left and top margins in centimetres

#### NVDA version:
2020.4

#### Windows version:
Windows
Edition	Windows 		10 Home
Version				1909
Installed on			17.02.2020
Operating system build		18363.1016

#### Does the issue still occur after restarting your computer?
Yes

**Comment by peterpacult** [comment]
The  Problem has also been reported to LibreOffice:
https://bugs.documentfoundation.org/show_bug.cgi?id=140732

**Comment by michaelweghorn** [comment]
This has just been implemented in the context of #11696 and the corresponding LibreOffice ticket, so this can probably be closed as a duplicate?

cc: @Adriani90

**Comment by Adriani90** [comment]
Thanks for picking this one, I am closing as it is a duplicate.
